examples/evaluate_lightning_document_classification.py

```diff
@@ -1,15 +1,24 @@
 import pprint
+from typing import Optional
 
 import typer
 
 from embeddings.defaults import RESULTS_PATH
 from embeddings.pipeline.lightning_classification import LightningClassificationPipeline
 from embeddings.utils.utils import build_output_path, format_eval_result
 
+LightningClassificationPipeline.DEFAULT_DATAMODULE_KWARGS.update(
+    {
+        "downsample_train": 0.01,
+        "downsample_val": 0.04,
+        "downsample_test": 0.04,
+    }
+)
+
```

Comment:
it that for development purposes? if so you can delete it

Reply:
removed